packages/nimble-components/rollup.config.js

```diff
@@ -2,6 +2,19 @@ import commonJS from '@rollup/plugin-commonjs';
 import resolve from '@rollup/plugin-node-resolve';
 import sourcemaps from 'rollup-plugin-sourcemaps';
 import { terser } from 'rollup-plugin-terser';
+import replace from '@rollup/plugin-replace';
+
```

Comment:
Could you describe these changes either in the PR description or an inline comment?

Also, if they affect the contents of all-components then the beachball change type should be `patch` instead of `none`.

Reply:
Updated.